eng/testing/performance/performance-setup.sh

```diff
@@ -267,24 +267,39 @@ else
     git clone --branch main --depth 1 --quiet https://github.com/dotnet/performance.git $performance_directory
     # uncomment to use BenchmarkDotNet sources instead of nuget packages
     # git clone https://github.com/dotnet/BenchmarkDotNet.git $benchmark_directory
+	if [ "$?" -ne "0" ]; then
+		echo "git clone failed with code $?"
+		exit 1
```

Comment:
Bash `if` conditional blocks are closed with `fi`. Can you fix up the additions to this file?